### DISCUSSION TITLE
Uncaught SyntaxError: expected expression, got '}'

### DISCUSSION THREAD

**Discussion by chapmanjacobd** [alpinejs/alpine#1652] [4. General]
After converting from Alpine v2 to v3 I got a cryptic error. 

It was because I had an empty `x-init=""`

So if you have one of those remove that and it might fix your issue :)

**Comment by danddanddand** [comment]
There is a PR to clean up these syntax errors https://github.com/alpinejs/alpine/pull/1607. I’ll test that case tomorrow to make sure it’s handled but I suspect it’s fixed.
